### DISCUSSION TITLE
Getting empty array of profile, after making GET request to `/profile`

### DISCUSSION THREAD

**Discussion by Dipikesh** [amzn/ads-advanced-tools-docs#87] [General]
I have integrated amazon ads authentication using LWA, successfully authenticated and generated access token, 
Now when i am making an get request to get the profile , I am getting empty array , but the profile exist on amazon ads account.
Why is it happening?

**Comment by jerirose** [comment]
Hi @Dipikesh , there are a few reasons you might see an empty array in the response. The most common reasons are:
- the advertising account is in a different region from the API host you have called
- the user account that created the authorization grant does not have **View and Edit** permissions for the advertising account

You can take a look at the tips in [the **Profiles response** section](https://advertising.amazon.com/API/docs/en-us/getting-started/retrieve-profiles#profiles-response) of our onboarding walkthrough, or see the information about regions and permissions in the [conceptual documentation on profiles](https://advertising.amazon.com/API/docs/en-us/concepts/authorization/profiles).

Please let us know if this doesn't help!

**Comment by jerirose** [comment]
Hi @Dipikesh , to follow up on this, some additional troubleshooting tips for an empty profiles list are available in [this article in Jira](https://amzn-clicks.atlassian.net/wiki/spaces/AA/pages/393266/Empty+profile+list) (requires Jira login).

I'll post the relevant tips not noted above in case you are unable to log in to the Jira support desk:

> - If the merchant has granted access via Seller Central User Permissions you'll need to make sure they gave you View/Edit permissions for Campaign Management; or
>     - Access to the API can be provisioned in the Ads Console by giving a user View/Edit access to Campaign Management.
> - For DSP Reporting: Users who find an empty profile list, but are trying to access DSP reporting may have their access provisioned via Manager Account.
>     - If so, DSP reporting does not require the use of a profile in order to access the reporting API.  Here, users should list their access by calling upon the managerAccount endpoint [as described here](https://advertising.amazon.com/API/docs/en-us/reporting/dsp/reporting-by-account-type#manager-account-advertiser-ids).

The support team recommends checking these troubleshooting steps before opening a support ticket via Jira.
